### PR TITLE
Polish up UI points on the edge

### PR DESCRIPTION
**Pull request by geofmureithi**
Improvements targeting the wizard.

### CODE REVIEW

File: app/src/Readme.tsx [hunk 1]
```diff
@@ -3,9 +3,35 @@ import { Navbar } from './Navbar'
...
+      <h2>Development</h2>
+      <p>
+        Nebula aims to provide the following functionality:
+        <ul>
+          <li>Compile contracts</li>
+          <li>Download compiled contracts</li>
+          <li>Deploy contracts</li>
+          <li>An IDE playground</li>
+          <li>
+            A contract management system via <code>#import</code>
```
Comment (eloylp) [inline app/src/Readme.tsx:19]: Would it be good to reference the [open discussion](https://github.com/eigerco/nebula/discussions/11) ?

File: app/src/Readme.tsx [hunk 2]
```diff
@@ -3,9 +3,35 @@ import { Navbar } from './Navbar'
...
+      <h2>What is it?</h2>
+      <p>Nebula is a tool for easing development of Soroban smart contracts.</p>
+      <h2>Development</h2>
+      <p>
+        Nebula aims to provide the following functionality:
+        <ul>
+          <li>Compile contracts</li>
+          <li>Download compiled contracts</li>
+          <li>Deploy contracts</li>
+          <li>An IDE playground</li>
```
Comment (eloylp) [inline app/src/Readme.tsx:17]: Are all of the above elements included in Nebula project ?

File: app/src/Readme.tsx [hunk 3]
```diff
@@ -3,9 +3,35 @@ import { Navbar } from './Navbar'
 
 export function Readme({ currentPage }: any) {
   return (
-    <div>
-      <Navbar currentPage="#/readme" />
-      Readme...
+    <div className="p-3">
+      <Navbar currentPage="#/" />
+      <h2>What is it?</h2>
+      <p>Nebula is a tool for easing development of Soroban smart contracts.</p>
```
Comment (eloylp) [inline app/src/Readme.tsx:9]: I would elaborate a bit more here. What are the parts we are going to see here ? contracts library + wizard ?

File: app/src/Readme.tsx [hunk 4]
```diff
@@ -3,9 +3,35 @@ import { Navbar } from './Navbar'
...
+          <li>Deploy contracts</li>
+          <li>An IDE playground</li>
+          <li>
+            A contract management system via <code>#import</code>
+          </li>
+        </ul>
+      </p>
+      <h2>Standardization</h2>
+      <p>
+        Nebula aims to provide standard interfaces for commonly used contacts.
```
Comment (eloylp) [inline app/src/Readme.tsx:25]: common interfaces and maybe default implementations ? should we specify that ?

File: app/src/Toolbox.tsx [hunk 5]
```diff
@@ -32,13 +30,19 @@ export function Toolbox({
...
           className="p-2"
           onChange={e => onContractTraitChanged(e.target.value)}
         >
           <option>Lottery</option>
           <option>Voting</option>
+          <option disabled>Auction</option>
+          <option disabled>MarketPlace</option>
+          <option disabled>FundMe</option>
+          <option disabled>Math</option>
+          <option disabled>PaymentSplit</option>
```
Comment (eloylp) [inline app/src/Toolbox.tsx:44]: Is this the full list of things or is this something provisional ?

**Review by eloylp — COMMENTED** [review]
I think this is still a draft PR in a very early stage, i started doing very abroad questions, probably because lack of more content or information regarding intentions. I think  its better to wait till this is still a bit more elaborated :)

File: app/src/Readme.tsx (hunk 3 above)
Comment (eloylp) [reply app/src/Readme.tsx:9]: I would also mention the intention of making it an UI pluggable module. I considered that a good idea.

File: app/src/Readme.tsx (hunk 2 above)
Comment (geofmureithi) [reply app/src/Readme.tsx:17]: `An IDE playground` will be Pulsar true, I will remove this.

File: app/src/Readme.tsx (hunk 3 above)
Comment (geofmureithi) [reply app/src/Readme.tsx:9]: FYI, You can suggest this way:
````
```suggestion
      <p>Nebula is a tool for easing development of Soroban smart contracts. (Some change anywhere)</p>
```
````

File: app/src/Toolbox.tsx (hunk 5 above)
Comment (geofmureithi) [reply app/src/Toolbox.tsx:44]: Is there a full list of things?

Comment (eloylp) [reply app/src/Toolbox.tsx:44]: I was thinking on this one we discussed in other channels. Not sure if want to put all of them.

Auction
Coin Flip
Lottery
MarketPlace
FundMe Contract
Raffle
Math (SafeMath + SafeDecimalMath)
Pause Contract - Allow pausing some transactional process
Payment splitter
reverse billing splitting
Voting facilities, create a smart contract that implements voting for different proposals, query the vote counting on each of them, etc ..

Comment (geofmureithi) [reply app/src/Toolbox.tsx:44]: This list is just temporary, I just summarized it to those easily understandable.

File: app/src/Readme.tsx (hunk 2 above)
Comment (geofmureithi) [reply app/src/Readme.tsx:17]: Removing one as you requested stops making sense. I am gonna remove all, and let you create a list. I am not sure what to write here.

**Comment by geofmureithi** [comment]
@eloylp I have made some changes please review.

**Review by eloylp — APPROVED** [review]
@geofmureithi i did a rebase with the main branch and added little changes to the readme, please check them.

Nitpick: The part of the playground is still a bit confusing for me. Could you add a description in the UI readme maybe ?

**Comment by geofmureithi** [comment]
Some checks are failing. I will

**Comment by geofmureithi** [comment]
What description do you mean?

**Comment by eloylp** [comment]
> What description do you mean?

image

Here we are describing the features, do you think we should mention something about the playground (as it appears in button of the wizard) ?

**Comment by geofmureithi** [comment]
I don't have any extra additions, I am gonna merge this and any new changes can be added later.

**Pull request merged**